examples/custom_speedup_methods.ipynb

```diff
@@ -0,0 +1,837 @@
+{
```

Comment:
This diff is large because I removed the embedded image, and instead put it in the google cloud bucket so it loads externally (large files should not be in the repo).